### ISSUE TITLE
Questions About Energy

### ISSUE THREAD

**Issue by YYX666660** [yxlu-0102/AP-BWE#12]
:) Another question about the **energy** amplitude of the reconstructed signal and gt signal. 
Here are one case of my test audios, extended log-amplitude spectrum has lower energy than the wideband log-amplitude spectrum. (so as some cases in VCTK)

<img width="1726" alt="Image" src="https://github.com/user-attachments/assets/e2374626-0136-43a3-9409-f3bd0e72a5a3" />

How can I alleviate this problem? Should I increase the loss hyper-parameters of the **amplitude spectrum loss**? Or any other  better methods?

**Comment by valentin710** [comment]
Just checking, your test audio seems to be compressed (mp3 or aac or sth similar) does your training data contain compressed data? if not, that might be the first thing I'd look at.

**Comment by YYX666660** [comment]
Well, the data processing of my training and testing are the same. The case above happens when the sampling ratio are large (16k to 48k/  8k to 48k),  24k to 48k doesn't have these problems.
